reviewrot/__init__.py

```diff
@@ -104,16 +111,40 @@ def get_arguments(cli_arguments, config_arguments, choices):
         else:
             parsed_arguments['ssl_verify'] = True
 
-
     if isinstance(parsed_arguments['ssl_verify'], str):
         parsed_arguments['ssl_verify'] = \
             expanduser(expandvars(parsed_arguments['ssl_verify']))
-        if not os.path.exists(parsed_arguments['ssl_verify'] ):
-            raise IOError("No certificate file found at %s" % parsed_arguments['ssl_verify'])
+        if not os.path.exists(parsed_arguments['ssl_verify']):
+            raise IOError("No certificate file found at %s "
+                          % parsed_arguments['ssl_verify'])
 
-    if parsed_arguments.get('insecure') and parsed_arguments.get('cacert', None):
+    if (parsed_arguments.get('insecure') and
+            parsed_arguments.get('cacert', None)):
         raise ValueError("Certificate file can't be used with insecure flag")
 
+    format = parsed_arguments.get('format')
+    if (format in ['oneline', 'indented'] and
+            parsed_arguments.get('show_last_comment') is not None):
```

Comment:
this could also be on a variable

Reply:
I assigned format to variable only because it is used more than once